src/frontend/test_runner/tests/testdata/join.yaml

```diff
@@ -47,36 +47,36 @@
     select t1.v1 as t1_v1, t1.v2 as t1_v2, t2.v1 as t2_v1, t2.v2 as t2_v2, t3.v1 as t3_v1, t3.v2 as t3_v2 from t1 join t2 on (t1.v1 = t2.v1) join t3 on (t2.v2 = t3.v2);
   batch_plan: |
     BatchExchange { order: [], dist: Single }
-      BatchHashJoin { type: Inner, predicate: $3 = $5, output_indices: all }
-        BatchExchange { order: [], dist: HashShard([3]) }
-          BatchHashJoin { type: Inner, predicate: $0 = $2, output_indices: all }
-            BatchExchange { order: [], dist: HashShard([0]) }
-              BatchScan { table: t1, columns: [v1, v2] }
-            BatchExchange { order: [], dist: HashShard([0]) }
-              BatchScan { table: t2, columns: [v1, v2] }
-        BatchExchange { order: [], dist: HashShard([1]) }
-          BatchScan { table: t3, columns: [v1, v2] }
+      BatchHashJoin { type: Inner, predicate: $5 = $6, output_indices: [0, 1, 2, 3, 4, 5] }
+        BatchExchange { order: [], dist: HashShard([5]) }
+          BatchHashJoin { type: Inner, predicate: $1 = $4, output_indices: all }
+            BatchExchange { order: [], dist: HashShard([1]) }
+              BatchScan { table: t1, columns: [_row_id, v1, v2] }
```

Comment:
why there's row_id in batch_query?

Reply:
BTW,  LogicalMultiJoin `as_reordered_left_deep_join` lost output_indices

Reply:
fixing